Cargo.toml

```diff
@@ -15,13 +15,14 @@ default = []
 protobuf = ["dep:prost", "dep:prost-types", "dep:prost-build"]
 
 [workspace]
-members = ["derive-encode"]
+members = ["derive-encode", "derive-register"]
```

Comment:
I guess we could as well do both encode and register in one derive crate? I am undecided. What is your rational for two crates?

Reply:
I just went off of the existing naming. I would be fine with one crate, but then I think it should be named just `prometheus-client-derive`. Also, should this be behind a feature so people don't have to compile syn if they don't need to?

Reply:
- I am in favor of one crate only.
- The rename to `prometheus-client-derive` sounds good.
- Moving `prometheus-client-derive` behind a feature sounds good to me.
- I suggest making it a default feature.